packages/kwordle/kwordle/main.css

```
@@ -0,0 +1,634 @@
+
+h1 {
+  text-align: center;
+  margin: 10px 0;
+  font-size: 24px;
+}
+
+#game-container {
+  padding-top: 10px;
+  width: 100%;
+  margin: 0 auto;
+  padding: 0 10px;
```

Biome (1.9.4)

[error] Unexpected shorthand property padding after padding-top



(lint/suspicious/noShorthandPropertyOverrides)